cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -157,58 +163,61 @@ body {
 
 .o-mega-menu {
 
-    &_heading {
-        // Override h2 element styles.
-        margin: 0;
-        font-size: 1em;
-    }
-
     // Reset default unordered list padding and margin.
     ul {
         padding-left: 0;
         margin-top: 0;
     }
 
     &_content {
+
         &-overview-heading {
             margin-bottom: 0;
-            font-size: unit( @base-font-size-px / @base-font-size-px, em );
+            font-size: unit( 18px / @base-font-size-px, rem );
             font-weight: normal;
         }
 
-        // Hide the ">" icon next to menu items on desktop…
-        &-link {
-            .cf-icon-svg {
-                display: none;
-            }
-        }
-
         // …however, show the icons in featured links.
-        &_content-list__featured {
+        &-item__has-icon {
             .cf-icon-svg {
                 display: inline-block;
             }
         }
 
-        // Featured menu content is link items with icons.
-        &-list__featured {
+        &-link,
+        &-overview-link {
+            width: 100%;
```

Comment:
This actually makes the desktop links wrap later, so the right-hand edge is too close sometimes I think, but we can adjust in another PR.

Reply:
Definitely too close

<img width="670" alt="Screen Shot 2020-03-19 at 11 41 26 AM" src="https://user-images.githubusercontent.com/704760/77085871-54cfa180-69d7-11ea-8b1d-e167e7f0ada8.png">

Reply:
There's also an overlap on certain screen sizes.

<img width="1185" alt="Screen Shot 2020-03-19 at 1 04 08 PM" src="https://user-images.githubusercontent.com/1060248/77095115-d4fc0400-69e3-11ea-8e74-6f3e42849dcd.png">

Reply:
You could calc it: `width: calc(100% - 5px);`

Reply:
Hmm, maybe. We want it 100% in some cases. I'll leave this one to another PR if you don't mind.